frontend/src/components/Stepper/Stepper.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import './Stepper.scss'
+
+const Stepper = (props) => {
```

Comment:
You can destructure the props here to make the code a bit cleaner and also to get a better overview of which props are used in the component i.e `const Stepper = ({ count, activeIndex }) => {` etc